x/committee/types/codec.go

```diff
@@ -47,7 +48,6 @@ func init() {
 	RegisterProposalTypeCodec(govv1beta1.TextProposal{}, "cosmos-sdk/TextProposal")
 	RegisterProposalTypeCodec(upgradetypes.SoftwareUpgradeProposal{}, "cosmos-sdk/SoftwareUpgradeProposal")
 	RegisterProposalTypeCodec(upgradetypes.CancelSoftwareUpgradeProposal{}, "cosmos-sdk/CancelSoftwareUpgradeProposal")
-
```

Comment:
Does `CommunityPoolLendWithdrawProposal` need to be registered here on the committee ModuleCdc?

Reply:
yeah it should be because `MsgSubmitProposal.GetSignBytes` uses the codec and contains a Proposal interface. I put together an explainer here: https://kava-labs.atlassian.net/wiki/spaces/ENG/pages/1237483521/Legacy+Amino+Codec+Registration

Reply:
ah, i was under the assumption that because they were registered in their respective modules in our code we didnt need to register them here. sounds like that is not the case.

does that mean that other committee proposals originating from our modules also need to be added? which as of right now is just `kavadist.CommunityPoolMultiSpendProposal`

Reply:
yeah probably worth adding those. I think it's not strictly a problem, as everything here is behind governance, but good to be consistent